sixtyfps_runtime/corelib/Cargo.toml

```diff
@@ -41,6 +41,7 @@ weak-table =  "0.3"
 scopeguard = "1.1.0"
 cfg-if = "1"
 unicode-segmentation = "1.8.0"
+# Note: When bumping this, make sure to adjust the rgb base url in ci.yaml for the nightly docs build
 rgb = "0.8.27"
```

Comment:
Could the CI extract this number with a simple shell script:
```
export RUSTDOCFLAGS="--extern-html-root-url rgb=https://docs.rs/rgb/`grep 'rgb = '  sixtyfps_runtime/corelib/Cargo.toml | sed 's/^.*"\(.*\)"/\1/' `/"
```

Or not worth it?

Reply:
Nice idea! I'll try that (after merging)